.circleci/config.yml

```diff
@@ -60,8 +57,8 @@ jobs:
       - *cache_node_modules
       - *persist_workspace
 
-  test_carbon:
-    <<: *container_config_carbon
+  test_fermium:
+    <<: *container_config_fermium
```

Comment:
Maybe replace `_carbon/_fermium` with just `_node`

Reply:
good point and I concur !